translations/cn/try_ruby_210.md

```diff
@@ -0,0 +1,17 @@
+---
+lang:   CN
+title:  太多逆置了
+answer: ^\["More still did (.+)
+load:   prev
+ok:     差不多是我们想要的了
+error:  
+---
+
+所以整首诗被倒过来，一个字母一个字母地倒过来。不过我真的想把这些行倒过来。
```

Comment:
A comment from Bitfox from the Ruby mailing list:

```
That's hard to understand for. it could be:

所以整首诗按照字母的顺序，被逐一倒置了。这显然不是我想要的。我期望按照行的顺序进行倒置，比如第一行变成最后一行。那么如下操作：

    poem.lines.reverse
```

Reply:
This sentence is indeed better than my translation 🤣